README.md

```diff
@@ -1,480 +1,133 @@
-# ![wgsl-import-spec](branding.svg)
+# A Community Standard for Enhanced WGSL
 
-# Summary
+We propose building an enhanced version of WebGPU's
+[WGSL](https://www.w3.org/TR/WGSL/) shading language
+using community tooling.
+We want to see WGSL enhancements shared more broadly.
+We want to enable a WGSL library ecosystem.
 
-We propose adding an importing mechanism to the WGSL shading language as an extension.
+Many community projects have independently uncovered
+a need for WGSL enhancements like module composition,
+conditional compilation,
+and runtime variable substitution.
```

Comment:
What do you mean with "runtime variable substitution"?

Reply:
I meant replacing wgsl text `#{BLOCK_SIZE}` with a value set at runtime, like `16`.

Need a better name tho, clearly. hmm.. maybe 'simple templating'?

Reply:
changed to simple templating.